perf-tests/clusterloader2/pkg/framework/config/client_config.go

```diff
@@ -35,16 +38,62 @@ const (
 	burst       = 200
 )
 
+var muxInitializedMultiClientConfig sync.Mutex
+
+var extraAPIServerAddresses []string
+
+func SetAPIServers(apiServerAddresses string) error {
+	muxInitializedMultiClientConfig.Lock()
+	defer muxInitializedMultiClientConfig.Unlock()
+	if len(extraAPIServerAddresses) > 0 {
+		return fmt.Errorf("Api Server addresses already set [%v]", extraAPIServerAddresses)
+	}
+
+	addrs := strings.Split(apiServerAddresses, ";")
+	for _, item := range addrs {
+		extraAPIServerAddresses = append(extraAPIServerAddresses, item)
```

Comment:
Why adding them one by one? extraAPIServerAddresses is exactly the same as addrs.

Reply:
Previously tried to validate each addresses. Will update.